__tests__/server/utils/onModuleLoad.spec.jsx

```diff
@@ -29,6 +29,7 @@ import { setCorsOrigins } from '../../../src/server/middleware/conditionallyAllo
 import { extendRestrictedAttributesAllowList, validateSafeRequestRestrictedAttributes } from '../../../src/server/utils/safeRequest';
 import { setConfigureRequestLog } from '../../../src/server/utils/logging/serverMiddleware';
 import { setCreateSsrFetch } from '../../../src/server/utils/createSsrFetch';
+import { configurePWA } from '../../../src/server/pwa';
```

Comment:
If you are mocking this, does it need to be imported?

Reply:
I imported it to assert that it was being called with the correct value, further below in the test.